sdk/identity/Azure.Identity/README.md

```diff
@@ -263,6 +263,11 @@ Not all credentials require this configuration. Credentials which authenticate t
 |`AZURE_USERNAME`|a username (usually an email address)
 |`AZURE_PASSWORD`|that user's password
 
+#### Managed identity
```

Comment:
Nit:
```suggestion
#### Managed identity

```

Not only recommended my markdownlint (you haven't installed that extension yet?!), but consistent throughout the doc.